src/Components/Navbar.jsx

```diff
@@ -0,0 +1,10 @@
+import { Link } from "react-router-dom";
```

Comment:
You could use conditionals and props to hide the signup and login button if they are logged in